mcp/common/src/main/java/org/springframework/ai/mcp/SyncMcpToolCallback.java

```diff
@@ -111,4 +112,10 @@ public String call(String functionInput) {
 		return ModelOptionsUtils.toJsonString(response.content());
 	}
 
+	@Override
+	public String call(String toolArguments, ToolContext toolContext) {
```

Comment:
I would like to understand why the default implementation of call(String, ToolContext) at ToolCallback is not enough as it handles null check and call(String) eventually.